README.md

```diff
@@ -52,26 +52,33 @@ where app.json may look something like:
             "sampleInterval": 1000
         }
     },
-    "plugins": {
-        "good": {
-            "opsInterval": 5000,
-            "requestHeaders": true,
-            "reporters": [{
-                "reporter": "good-console",
-                "events": { "response": "*", "ops": "*", "log": "*", "error": "*" }
-            },
-            {
-                "reporter": "good-file",
-                "events": { "response": "*", "error": "*" },
-                "config": "/log/response.log"
-            },
-            {
-                "reporter": "good-file",
-                "events": { "ops": "*" },
-                "config": "/log/ops.log"
-            }]
+    "registrations": [
```

Comment:
Yes, I think it would be a good idea to simplify the manifest (I don't know what was wrong with just plugins vs registrations before but rejoice just implements glue manifests and this is how it is currently being implemented with 3.x.